e2e/support/helpers/e2e-embedding-helpers.js

```diff
@@ -115,28 +115,36 @@ export function visitIframe() {
   });
 }
 
-export function openToEmbedModal({
-  isAdmin = true,
-  isPublicSharingEnabled = true,
-} = {}) {
+export function openPublicLinkDropdown() {
```

Comment:
I just noticed the name. Isn't it more correct to be a popover? A dropdown is an input where you could select usually one option from multiple options.
```suggestion
export function openPublicLinkPopover() {
```

Also when paring with the below function `openEmbedModalFromMenu` this could be made more aligned

```suggestion
export function openPublicLinkPopoverFromMenu() {
```

Reply:
Perfect 😄